frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -88,20 +88,22 @@
         </template>
 
         <template #item.actionButtons="{ item }">
-          <v-tooltip v-if="!isApplicationDownloadable(item)" content-class="tooltip" text="This PDF will be generated when the application is approved">
-            <template #activator="{ props }">
-              <v-btn v-bind="props" variant="text">
-                <v-icon :disabled="true" icon="fa:fa-regular fa-file-pdf"></v-icon>
-              </v-btn>
-            </template>
-          </v-tooltip>
-          <v-btn v-else variant="text" @click="downloadPDF(item)">
-            <v-icon icon="fa:fa-regular fa-file-pdf"></v-icon>
-          </v-btn>
-
           <v-btn v-if="isApplicationCancellable(item)" variant="text" @click="toggleCancelDialog(item)">
             <v-icon icon="fa:fa-regular fa-trash-can"></v-icon>
           </v-btn>
+
+          <div v-else>
```

Comment:
instead of v-else, I think we should have a separate computed value for showDownloadPDFButton. because if the isApplicationCancellable logic has Permission validation, I think your download button will display if the user has no permission.

Reply:
that's a good point.. I will add in a computed value that checks for View Applications permission